test/image-prepare

```diff
@@ -142,6 +142,43 @@ def build_install_ostree(dist_tar, image, verbose, quick):
     return args
 
 
+def build_install_container(dist_tar, image, verbose, quick):
+    """Build VM with cockpit/ws container
+
+    This can test an image with beibooting from a cockpit/ws container, without
+    any installed cockpit packages.
+    """
+    # this needs to be the image that corresponds to the actual cockpit/ws container
+    # parse from containers/ws/Dockerfile
+    with open(os.path.join(BASE_DIR, "containers/ws/Dockerfile")) as f:
+        for line in f:
+            if line.startswith("FROM"):
+                fedora_version = 'FROM registry.fedoraproject.org/fedora:40 AS builder'.split('fedora:')[1].split()[0]
```

Comment:
I think this is supposed to be `line.split()`?

Reply:
:monkey_face: thanks for spotting! fixed (I accidentally copied too much from my interactive Python REPL where I tested the string mangling)